src/app/rpmostree-builtin-status.cxx

```diff
@@ -369,6 +388,20 @@ print_daemon_state (RPMOSTreeSysroot *sysroot_proxy,
         }
     }
 
+  g_autofree char *update_driver_sd_unit = NULL;
+  if (!get_update_driver_sd_unit (&update_driver_sd_unit, error))
+    return FALSE;
+
+  if (!update_driver_sd_unit)
+    {
+      if (opt_verbose)
+        g_print ("UpdateDriver: no update drivers registered\n");
+    }
+  else
+    {
+      g_print ("UpdateDriver: %s\n", update_driver_sd_unit);
```

Comment:
Hmm, should this be printed as part of the `AutomaticUpdates` field above instead? Something like:
- if there's an update driver: `AutomaticUpdates: driven by $name ($systemd unit)
- if rpm-ostree's automatic updates are enabled: `AutomaticUpdates: $policy`
- if rpm-ostree's automatic updates are disabled: `AutomaticUpdates: disabled`

?

Reply:
I wasn't sure if they were totally mutually exclusive. But if they are, this would make sense.

Reply:
They are at a conceptual level, but I think we should probably harden this in the code as well in a follow-up. E.g. if during an `rpm-ostreed-automatic.service` trigger we detect that there is a registered update driver, we should probably log something and exit early?